src/test/resources/PostgresAppTestExtensionTest/test-config.yml

```diff
@@ -0,0 +1,8 @@
+server:
+  type: simple
+  connector:
+    type: http
+    port: 10000
+
+database:
+  driverClass: org.postgresql.Driver
```

Comment:
I guess might as well make GitHub happy and add the newline here, and also for the migrations file.

Reply:
Done